common/changes/@microsoft/rush/ianc-per-project-shrinkwrap_2019-09-13-20-41.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "When using pnpm, introduce the ability to only rebuild projects whose direct and/or indirect dependencies have changed.",
```

Comment:
The logic here sounds weird, since it seems to exclude non-dependency files.

Maybe instead:  `Improve "rush build" to avoid rebuilding unnecessarily when the package-lock.json file has changed (pnpm only)`